packages/webpack-exp-plugin/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "@bem-react/webpack-exp-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
```

Уже есть конфиг с настройками, давай сделаем аналогично, чтобы в CI запускалось

```suggestion
    "unit": "../../node_modules/.bin/jest --config ../../.config/jest/jest.config.js"
```